plugins/org.jkiss.dbeaver.ext.postgresql.ui/src/org/jkiss/dbeaver/ext/postgresql/ui/editors/PostgreSourceViewEditor.java

```diff
@@ -17,20 +17,35 @@
 
 package org.jkiss.dbeaver.ext.postgresql.ui.editors;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.eclipse.core.runtime.CoreException;
```

Comment:
The error appears due to internal java imports being first in PostgreSourceViewEditor